src/Events/Configuration/AuthorizationConstants.cs

```diff
@@ -9,5 +9,10 @@ public static class AuthorizationConstants
         /// Id for policy scope for allowing publishing of events
         /// </summary>
         public const string POLICY_SCOPE_EVENTS_PUBLISH = "ScopeEventsPublish";
```

Comment:
Should we keep these two scope names consistent somehow?

Reply:
Discussed and keeping as is for now. Standard across projects. Will need to discuss further how a new structure should look